README.md

```diff
@@ -0,0 +1,140 @@
+let headers = {
+    'Content-Type': 'application/json'
+};
+let auth = {
+    user: 'DummyUser',  // Authentication user
+    pass: 'DummyPassword',  // Authentication password
+    sendImmediately: true   // Optional (defaults to true) Flag to determine whether the request should send the basic authentication header along with the request.
+                                          // Upon being false, Request will retry with a proper authentication header after receiving a 401 response from the server
+}
+let url = 'https://dummyurl.com/secureservice'
+let response = yield I.sendGet(url, headers, auth);
+```
+
+2. **_POST requests_**
+```js
+let response = yield I.sendPost(URL, HEADERS, REQUEST BODY OR PAYLOAD);
+```
+The POST request will insert information into the endpoint and store any response from the server within the "response" object
+1. **REQUEST BODY or PAYLOAD** _(object)_: This is the data you wish to post.
+```js
+let headers = {
+    'Content-Type': 'application/json'
+};
+let payload = {
+    title: 'Dummy book title',
+    author: 'Dummy author name'
+};
+let url = 'https://dummyurl/books';
+let response = yield I.sendPost(url, headers, payload);
+// This request will add a new book with the "title" and "author" described within the "payload" object
+```
+
+3. **_PUT requests_**
+```js
+let response = yield I.sendPut(URL, HEADERS, REQUEST BODY OR PAYLOAD);
+```
+The PUT request will update an object determined within the URL and it will use the information within the payload. It will also store any response from the server within the "response" object
+```js
+let headers = {
+    'Content-Type': 'application/json'
+};
+let payload = {
+    title: 'An updated book title',
+    author: 'Updated author name'
+};
+let url = 'https://dummyurl/books/1';   // This will indicate that the updated object should be the one with id = 1
+let response = yield I.sendPut(url, headers, payload);
+// This request will update book with id = 1 with the new "title" and "author" described within the "payload" object
+```
+
+4. **_DELETE requests_**
+```js
+let response = yield I.sendDelete(URL, HEADERS);
+```
+The DELETE request will remove an object determined within the URL and it will store any response from the server within the "response" object
+```js
+let headers = {
+    'Content-Type': 'application/json'
+};
+let url = 'https://dummyurl/books/1';   // This will indicate that the removed object should be the one with id = 1
+let response = yield I.sendPut(url, headers, payload);
+// This request will remove book with id = 1 from the endpoint
+```
+
+### RESPONSE object
+***
+The response object contains valuable information depending the method of the request previously sent.
+For example:
+* Response code for the request (important to know the result of sending the request to the server)
+* Response body (a complex object containing all the information about the response, mainly important for a GET request)
+
```

Comment:
Mention we used **unirest** library to execute the calls and there they can see how this object is built.